src/classes/Trip.js

```diff
@@ -4,7 +4,36 @@ class Trip {
     this.userTrips = tripData.filter(trip => trip.userID === userID);
     this.trips = tripData;
     this.destinations = destinationData;
-  }
-}
+  };
+
+  getTripStatus(tripStatus) {
+    const filteredTrips = this.userTrips
+      .filter(trip => trip.status === tripStatus);
```

Comment:
Empty line before return

Reply:
I have added the white space in the getTripStatus method and also added an additional method to get the total cost for a given year